packages/odyssey-storybook/src/contributing/Documentation.stories.mdx

```diff
@@ -0,0 +1,207 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Contributing/Documentation" />
+
+# Documentation
+
+In order to be considered for release, components require documentation. We provide a template for component documentation to make this process easier and ensure consistent docs.
+
+## Doc style
+
+Placeholder for guidelines surrounding:
+
+- Writing copy within documentation (e.g. sentence casing, example format)
+- Guide to writing CSF stories vs inline MDX stories
+- Use SB tables unless you are documenting an Odyssey table
+
+## Doc sections
+
+From top to bottom, component documentation should include these sections. Optional content is marked as such.
+
+<dl>
+  <dt>Overview</dt>
+  <dd>
+    This section should be named after the component and include a short
+    description as well as links to the Figma files and source code.
+  </dd>
+  <dt>Anatomy</dt>
+  <dd>
+    This section houses a figure which provides a visual breakdown of the
+    component parts, layout, and tokens.
+  </dd>
+  <dt>Arguments / Props</dt>
+  <dd>
+    This section should be automatically populated by an
+    <code>&lt;ArgsTable/&gt;</code>.
+  </dd>
+  <dt>Behavior (Optional)</dt>
+  <dd>
+    While Usage section below describes guidelines for designers and engineers,
+    Behavior describes default behaviors included with components.
+  </dd>
+  <dd>
+    This section is not required if the component has limited or self-evident
+    behavior.
+  </dd>
+  <dt>Variants (Optional)</dt>
+  <dd>
+    This section decribes the differing configurations available for the
+    component.
+  </dd>
+  <dd>
+    These variants could be global or semantic changes (e.g. "Success" vs
+    "Danger") or different component assemblies (e.g. "With actions")
+  </dd>
+  <dt>States</dt>
+  <dd>This section covers all the states a component can find itself in.</dd>
+  <dd>
+    Pseudo-classed states requiring user interaction, specifically `:focus` and
+    `:hover` are not required if those states can be observed via quick user
+    interaction.
+  </dd>
+  <dt>Usage</dt>
+  <dd>
+    This is where we lay out the function of components and how they can be used
+    within a design/view/page.
+  </dd>
+  <dd>DOs and DONTs are appropriate for this section.</dd>
+  <dt>Content Guidelines</dt>
+  <dd>
+    This is where we lay out any requirements around acceptable content for use
+    within the component.
+  </dd>
+  <dd>
+    This includes both what a component may contain as well as how that content
+    should be written.
+  </dd>
+  <dd>DOs and DONTs are appropriate for this section.</dd>
+  <dt>Accessibility</dt>
+  <dd>
+    This section includes any guidelines required for compliance as well as
+    design rationale already leveraged to ensure compliance - both "please do
+    this" and "we do this because".
+  </dd>
```

Comment:
I know this isnt a broader contribution guide, but it may be worth pointing out things like the WAI-ARIA style guide as a resource for what good a11y docs look like. Maybe this isn't the right place for that though?

Reply:
I think this is the right place (or page at least), but I'm not sure if we want folks to copy that style for this use case.

To my mind, theirs are "how to meet criterion" generally and ours are "things to pay attention to when using this component".

Do you have an example of something you think would be worth emulating?